README.md

```diff
@@ -107,4 +107,4 @@ module.exports = [
   ...testsWithFiles,
   ...fpConfig
 ]
-```
\ No newline at end of file
+```
```

Comment:
I didn't change this line, I guess GH readme editor changed the line ending or something